resources/assets/js/utils.js

```diff
@@ -108,21 +112,13 @@ export const imageValidator = (inputFile, rules = []) => {
         let size = bytesToMegabytes(image.size);
 
         if (size > target) {
-            pushError(
+            errorBag.add(
                 getCallerName(),
                 `The image should be no bigger than ${target}MB`
```

Comment:
I think we should validate the size after the compression don't you think? think part of the idea of this compression is allowing the user to upload a bigger image but compress it by them.

If so, the error message could be something like "Event after compressed the image is bigger than ${target}x${target}px"  (can be discussed on slack, but you got the idea)

Reply:
@alfonsobries I left the size check before the compression because from the user perspective, he doesn't know that the platform compresses images. In my opinion adding a limit upfront fixes boundaries for the user and lets him known what he is doing.
Let's say that a user can upload an image up to 20Mb, then the platform compresses the image and the result is 7Mb. If we allow only 6Mb this image will fail the validation and the user won't know how much he needs to resize the image to pass the validation and I think this can be frustrating.
What do you think?

Reply:
based on discussion in slack, it might help if we increase the limits to 5MB and check how well it would compress a png/jpg of that size to see what remains. That way users have a bit more space to work with